packages/transaction-controller/src/utils/layer1-gas-fee-flow.ts

```diff
@@ -46,17 +46,23 @@ export async function updateTransactionLayer1GasFee(
  * @param messenger - The messenger instance.
  * @returns The layer 1 gas fee flow for the transaction, or undefined if none match.
  */
-function getLayer1GasFeeFlow(
+async function getLayer1GasFeeFlow(
```

Comment:
Is this type exported? In case it is, we should mark these changes as BREAKING, detailing why that's the case in the CHANGELOG